actions/code-release/action.yml

```diff
@@ -55,87 +40,42 @@ runs:
         id: version
         uses: notiz-dev/github-action-json-property@release
         with: 
-           path: '${{inputs.path-to-json}}'
+           path: '${{inputs.json-path}}'
            prop_path: 'version'
-      - name: Install NPM dependencies and create folder with files used in subsequent steps
-        shell: bash
-        run: | 
-          if [[ "${{inputs.repository-type}}" = "typescript" ]]; then
-            if [ -n "${{inputs.nested-folder}}" ]; then
-                 cd ${{inputs.nested-folder}}
-                 fi
-            if [ -n "${{inputs.exclude-files}}" ]; then
-                 npm install
-                 rsync -arv ${{inputs.exclude-files}} . ./${{inputs.source-directory}}
-                 fi
-            if [ -n "${{inputs.included-files}}" ]; then
-                 npm install
-                 tsc
-                 rsync -arv ${{inputs.included-files}} ./${{inputs.source-directory}}
-                 fi
-            exit 0
-          elif [[ "${{inputs.repository-type}}" = "python" ]]; then
-            if [ -n "${{inputs.nested-folder}}" ]; then
-                 cd ${{inputs.nested-folder}}
-                 fi
-            rsync -arv ${{inputs.exclude-files}} . ./relesae
-            exit 0
-          elif [[ ${{inputs.repository-type}} = "layer" ]]; then
-            exit 0
-          else
-            exit 1
-          fi
-      - name: Archive files for S3 package
-        uses: thedoctor0/zip-release@master
-        with:
-          type: 'zip'
-          directory: ./${{inputs.source-directory}}
-          filename: '${{inputs.repository-name}}-${{steps.version.outputs.prop}}.zip'
-      - name: Creat txt file with version number for S3 package
+      - name: Run schell script
```

Comment:
Maybe smth like `Create a zip package` would be more descriptive.

Reply:
Changed